protoc-gen-tstypes/testdata/output/outpattern-2/example/example1.d.ts

```diff
@@ -11,11 +11,16 @@ declare namespace example {
         PRODUCTS = "PRODUCTS",
         VIDEO = "VIDEO",
     }
+    export interface SearchRequest_XyzEntry {
+        key?: string;
+        value?: number;
+    }
+
     export interface SearchRequest {
         query?: string;
         pageNumber?: number;
         resultPerPage?: number;
-        corpus?: Corpus;
+        corpus?: SearchRequest_Corpus;
         sentAt?: google.protobuf.Timestamp;
         xyz?: { [key: string]: number };
```

Comment:
Shouldn't the type for `xyz` be like this instead?
```
xyz?: SearchRequest_XyzEntry;
```

Reply:
Hm yeah good call. I’ll open a separate PR for that (or you can).